contracts/interfaces/IBosonRouter.sol

```diff
@@ -36,6 +36,19 @@ interface IBosonRouter {
         uint256[] calldata metadata
     ) external;
 
+    function requestCreateOrderTKNTKNWithPermitConditional(
```

Comment:
As mentioned before, I guess it seems like we should really be adding in the ability to create all of the XXXYYY variants to the BosonRouter ...

Reply:
done